src/shared/Constants.ts

```diff
@@ -0,0 +1,11 @@
+// Auth container
+export const AUTH_CMWL_TITLE = 'Cromwell Pipeline';
+export const AUTH_EMAIL_LABEL = 'Email';
+export const AUTH_PASSWORD_LABEL = 'Password';
+export const AUTH_SIGN_IN_TEXT = 'Sign in';
+export const AUTH_COPYRIGHT_TEXT = 'Copyright ©';
+
+// Dashboard container
+export const DASHBOARD_CMWL_TITLE = 'Cromwell Pipeline';
+export const DASHBOARD_PROJECTS_NAV_PRIMARY = 'Projects';
+export const DASHBOARD_EXECUTIONS_NAV_PRIMARY = 'Executions';
```

Comment:
I like that you moved common texts into a separate module. I've also heard that there are plans for localisation. As a suggestion — you can do quite easy localisation provider using React contexts in combination with hooks ;)

Reply:
I've thought about [i18next](https://www.i18next.com/) but it would be more useful (considering this as learning purposes) to implement it by ourselves.

P.S. I'll investigate React deeper contexts but the first thought was about "Hmm... If we have the React contexts why I'm using Redux?"

Reply:
The localization provider was implemented :)